js/client.ts

```diff
@@ -4,11 +4,13 @@ type DOM = {
 } | {
   type: 'leaf',
   attrs: { [key: string]: string | null },
-  element: string
+  element: string,
+  namespace?: string
```

Comment:
If you define `namespace: string | null` you'll be able to simplify the `ToJSON` instance [here](https://github.com/pkamenarsky/replica/pull/13/files#diff-faf6b2e2f6b19fbbf6bc2733462bbea4R36) without having to change the ts/js code.

(It'll always send an additional `namespace` field over the wire, but payload optimisation isn't a priority at the moment and must be handled in other ways anyway).

Reply:
Sounds great!